reference/call-home.md

```diff
@@ -0,0 +1,91 @@
+# Call-home metrics 
+
+ By default, Mayastor collects some basic information related to the number and scale of user-deployed instances. The collected data is anonymous and is encrypted at rest. This data is used to understand storage usage trends, which in turn helps maintainers prioritize their contributions to maximize the benefit to the community as a whole. 
+
+{% hint style="info" %} 
+
+No user-identifiable information, hostnames, passwords, or volume data are collected. **ONLY** the below-mentioned information is collected from the cluster. 
+
+{% endhint %} 
+
+
+A summary of the information collected is given below.
+
+| **Cluster information** | 
+
+| :--- | 
+
+|**K8s cluster ID**: This is a SHA-256 hashed value of the UID of your Kubernetes cluster's `kube-system` namespace.| 
+
+|**K8s node count**: This is the number of nodes in your Kubernetes cluster.| 
+
+|**Product name**: This is the name given during Helm install. | 
```

Comment:
@GlennBullingham , @niladrih Just wondering, do we really need to collect "Product name" as part of call-home data?

Reply:
The reporting module likely uses this to separate between Mayastor and Bolt.

The Product name and K8s Cluster ID are in the headers also, but because our reporting module code only gets access to the bucket contents, we can't likely have logic attached to headers. @datacore-vvarakantham can correct me if I am wrong here.

If we own the server code someday, these 2 fields need not be duplicated in the body.

Reply:
On second thought, we may choose to default to recognise Mayastor by the absence of the 'Product name'.

Reply:
- Those fields should absolutely be duplicated in the body, as should all other "important" fields.  Headers are only available within/to Swarm.  If the JSON content is exported, or passed to other components, how else would we have that information?  (as Niladri has said)
- "Product Name" is required but should be taken by the same means as by which we are taking the production version information.  The user could supply any name during Helm install, and that will prevent us from correctly identifying what is installed if we use that arbitrary value.

Reply:
The Product Name is hard coded. It is not the helm chart release name.

Reply:
Good.  Thanks for the clarification.

Reply:
Ok, looks like we are intending the Product Name to be "Mayastor" here, not a user-configurable value. Then, it's fine.